crates/lang/codegen/src/generator/metadata.rs

```diff
@@ -275,11 +275,9 @@ impl Metadata<'_> {
                         as ::ink_lang::reflect::TraitMessageInfo<#local_id>>::SELECTOR
                 }};
                 let ret_ty = Self::generate_return_type(message.output());
+                let label = [trait_ident.to_string(), message_ident.to_string()].join("::");
```

Comment:
Do you prefer this logic here or in `from_trait_and_label` on "spec" level?

What do you think about the idea that the namespace will affect the label too(not only selector)? If you agree then I will implement it like this https://github.com/paritytech/ink/blob/master/crates/lang/ir/src/ir/selector.rs#L108

Reply:
I meant, that I removed `from_trait_and_label ` from `spec` and implement the logic of concatenating(joining of trait name and method name by "::") here, on `generator` level.

Reply:
Alright, got it. It's fine here.

Do you want to implement the namespace affecting the label as well? Could also be done as a follow-up.

Reply:
I'll merge this now, I think the PR is fine in its current state and we've been putting you on hold for way too long :-).

Feel free to create a follow-up for including the namespace into the label.

Reply:
I can implement that in follow-up PR=) @Robbepop What do you think about the namespace affecting the label?